shared/validation/languages/it.js

```diff
@@ -0,0 +1,47 @@
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration. 
+// Italian: finché non abbiamo il controllo sui caratteri, o quello che tiene conto della presenza di numeri, metto 17 parole che dovrebbero essere salvo rarissimi casi sempre entro il limite di 125 caratteri. 
+const MAX_WORDS = 17;
+
+// Numbers that are not allowed in a sentence depending on the language. For
+// Italian: tutti i numeri sono consentiti ma se e quando avremo la possibilità di fare regole personalizzate metteremo una condizione ulteriore sulla lunghezza se vi sono dei numeri nella frase.
+const NUMBERS_REGEX = /[0-9]+/;
+
+/* eslint-disable-next-line no-useless-escape */
+// Italian: Simboli non permessi, aggiungere anche qui sotto oltre che nella regex:
+// “ ” ‘ ’ ( ) É 
+//doppio " " e più di un "." nella stessa frase.
+const SYMBOL_REGEX = /[“”‘’()É]| {2,}|\..*\./;       //  original:   /[<>\+\*\\#@\^\[\]\(\)\/]/;
```

Comment:
can you include the missing symbols from the original as well? We certainly don't want to allow `@` or `#` for example. Whitespaces are handled in the cleanup, and if you really want to include whitespaces here, please use `/s`.